src/ch02-09-03-erc20-apibara.md

```diff
@@ -0,0 +1,520 @@
+<img alt="ERC20 Apibara Main" src="img/ch02-apibara-erc20-main.png" class="center" style="width: 100%;" />
+
+# ERC-20 UI with Apibara-react and StarknetKit
+
+In this section, we will be exploring how to build a web3 application with [Apibara](https://github.com/apibara/starknet-react) react library, [StarknetKit](https://www.starknetkit.com/docs/getting-started), and an ERC-20 smart contract written in the [Cairo](https://book.cairo-lang.org/title-page.html) language. This tutorial is similar to [ERC-20 UI](ch02-10-01-erc20-ui.md) tutorial but with the addition of utilizing Apibara-react, StarknetKit and up to date versions of the tools and libraries.
+
+## Prerequisites
+
+These are the main tools we will be using in this section
+
+- Scarb v2.6.4 with Cairo v2.6.3
+- Starkli v0.2.8
+- Openzeppelin library v0.10.0
+- @starknet-react/chains v0.1.0
+- @starknet-react/core v2.3.0
+- get-starknet-core v3.2.0
+- starknet v5.29.0
+- starknetkit v1.1.4
+- NodeJS v18.19.1
+- NextJS v14.0.2
+- Visual Studio Code (or your favorite IDE!)
+
+Before we start, this guide assumes the reader is familiar in the following:
+
+1. Cairo
+2. ReactJS/NextJS
+3. Declaring/deploying Starknet contracts
+4. Usage of blockchain explorers like [Voyager](https://voyager.online/)
+5. Usage of Starknet wallets like **Argent** or any blockchain wallets
+
+We will first start with building the contract.
+
+**[IMPORTANT]** *Before we start building the contract, make sure that you have your environment setup by clicking [here](https://docs.starknet.io/documentation/quick_start/environment_setup/) and navigate to this github [**repo**](https://github.com/FriendlyLifeguard/starknet_erc20_example), clone it and follow the instruction on the README to setup the project.*
```

Comment:
Here, you have to copy your github project and past it in this section
https://github.com/starknet-edu/starknetbook/tree/main/examples

Similar to the other examples/tutorials